scripts/migration/migrate_primary_institutions.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import logging
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from scripts import utils
+from website.app import init_app
+from website.models import Node, NodeLog
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.WARN)
```

Comment:
Why is this here? `INFO` level logs shouldn't be removed, and there isn't any error handling in this script that would merit a `WARN` log